openaerostruct/tests/test_aero_morphing_multipoint.py

```diff
@@ -1,4 +1,4 @@
-from openmdao.utils.assert_utils import assert_rel_error
+from openmdao.utils.assert_utils import assert_near_equal
```

Comment:
I personally prefer the asserts from `numpy`, i.e. `np.testing.assert_all_close`, since it's good for both large and small numbers. But if the math behind this is okay then I'm fine with that also.

Reply:
`assert_near_equal` checks relative errors so it should be good I think.